yourgov/components/StaffMembersTable/DataTable.tsx

```diff
@@ -0,0 +1,297 @@
+							<TableCaption>
+								{caption}
+								<VisuallyHidden>
+									, column headers with buttons are sortable.
+								</VisuallyHidden>
+							</TableCaption>
+						)}
+						<TableHead>
+							<TableRow aria-rowindex={1}>
+								{selectable && (
+									<TableHeader scope="col" width="5rem">
+										Select
+									</TableHeader>
+								)}
+								{headers.map(
+									({ isSortable: isFieldSortable, label, sortKey, width }) => {
+										if (isTableSortable && isFieldSortable) {
+											const isFieldTheActiveSortField = sort?.field === sortKey;
+											const onClick = () =>
+												setSort?.({
+													field: sortKey,
+													order:
+														sort?.field === sortKey && sort?.order === 'ASC'
+															? 'DESC'
+															: 'ASC',
+												});
+
+											return (
+												<TableHeaderSortable
+													key={sortKey}
+													width={width}
+													sort={
+														isFieldTheActiveSortField ? sort?.order : undefined
+													}
+													onClick={onClick}
+												>
+													{label}
+												</TableHeaderSortable>
+											);
+										}
+
+										return (
+											<TableHeader key={sortKey} scope="col" width={width}>
+												{label}
+											</TableHeader>
+										);
+									}
+								)}
+							</TableRow>
+						</TableHead>
+						<TableBody>
+							{loading ? (
+								<Fragment>
+									{Array.from(Array(pagination.perPage).keys()).map((i) => (
+										<TableRow key={i}>
+											{selectable && (
+												<TableCell>
+													<SkeletonText />
+													<VisuallyHidden>Loading</VisuallyHidden>
+												</TableCell>
+											)}
+
+											<TableCell>
+												<SkeletonText />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+
+											<TableCell>
+												<SkeletonText />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+
+											<TableCell>
+												<SkeletonText />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+
+											<TableCell>
+												<SkeletonText />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+
+											<TableCell>
+												<SkeletonText />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+
+											<TableCell>
+												<SkeletonBox height={32} />
+												<VisuallyHidden>Loading</VisuallyHidden>
+											</TableCell>
+										</TableRow>
+									))}
+								</Fragment>
+							) : (
+								<Fragment>
+									{data.map((item) => {
+										const {
+											index,
+											name,
+											role,
+											status,
+											lastActive,
+											dateJoined,
+										} = item;
+
+										// Adding 2 because the table header row is the first row
+										const rowIndex = index + 2;
+										const id = name.replace(' ', '-');
```

Comment:
I would do this to support any possible "3 word names" just in case
```suggestion
										const id = name.replace(/\s/g, '-');
```

Reply:
Hmmm I wonder if it's worth adding real ids to the data, since you're reversing it in DataTableBatchActionsBar.tsx and would also need to do the same on a `/staff/[slug]` view page